dev/04_transform.ipynb

```diff
@@ -864,6 +864,22 @@
     "        return x"
```

Comment:
<p>Note that you shouldn't return anything: the InplaceTransform does it for you.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_dev/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>